src/twisted/python/randbytes.py

```diff
@@ -109,11 +109,11 @@ def insecureRandom(self, nbytes):
         @return: a string of random bytes.
         @rtype: C{str}
         """
-        for src in ("_randBits", "_randModule"):
-            try:
-                return getattr(self, src)(nbytes)
-            except SourceNotAvailable:
-                pass
+        try:
```

Comment:
I guess this is a drive-by and not related to type checking.

Reply:
This is required for the method to type check